test/parallel/test-crypto-sign-verify.js

```diff
@@ -116,7 +116,7 @@ const keyPem = fs.readFileSync(common.fixturesDir + '/test_key.pem', 'ascii');
         key: keyPem,
         padding: crypto.constants.RSA_PKCS1_OAEP_PADDING
       });
-  }, /^Error: padding must be RSA_PKCS1_PADDING or RSA_PKCS1_PSS_PADDING$/);
+  }, /^Error:.*illegal or unsupported padding mode$/);
```

Comment:
Can you add test cases for obviously invalid inputs (null, undefined, NaN, "boom", an object, etc.)?